src/managednetworkfabric/azext_managednetworkfabric/aaz/latest/networkfabric/fabric/__init__.py

```diff
@@ -9,11 +9,13 @@
 # flake8: noqa
 
 from .__cmd_group import *
+from ._commit_configuration import *
 from ._create import *
 from ._delete import *
 from ._deprovision import *
 from ._list import *
 from ._provision import *
 from ._show import *
 from ._update import *
+# from ._validate_configuration import *
```

Comment:
Why this line is commented out? If you don't want this command, you don't need to generate it.

Reply:
I have removed it @kairu-ms. Could you please review the updated changes.